### ISSUE TITLE
Drop `language` flag

### ISSUE THREAD

**Issue by rafaelrinaldi** [rafaelrinaldi/mdn#7]
Since we only support JavaScript (the default) and CSS, I think we can do:

``` sh
$ mdn background-image --css
```

Instead of

``` sh
$ mdn background-image --language=css
```

We can of course introduce this change with backwards compat.

**Comment by greg-js** [comment]
+1

Makes a lot of sense, but of course let's keep it backwards compatible, no need to get rid of the flag altogether

**Comment by greg-js** [comment]
Addressed in #5 and #9
